coalib/bearlib/languages/__init__.py

```diff
@@ -6,7 +6,6 @@
 from .Language import Language
 from .Language import Languages
 
-from .definitions.Unknown import Unknown
```

Comment:
unknown is intentionally first.  it isnt a language.

Add a blank line under it to make this more obvious.